### DISCUSSION TITLE
Erreur 404

### DISCUSSION THREAD

**Discussion by vanouzbek** [bokub/conso-api#7] [Aide]
Bonjour, 
Je débute sur Home assistant et j'aurais aimais ajouter ma conso éléc mais lorsque je suis le lien sur https://conso.boris.sh/ : "j'accède à mon espace client Enedis" je tombe sur une erreur 404. J'imagine que le lien d'Enedis à sauter mais peut être un souci chez moi. 
Ci-joint un screen de la page directement après avoir cliqué sur le lien : ( https://drive.google.com/file/d/1BU0uSwtXFzrkoogAeic79rUaRon6H_9V/view?usp=sharing )

D'avance merci pour vos réponses :]

**Comment by bokub** [comment]
C'est simplement Enedis qui est KO, comme expliqué dans #6 :slightly_smiling_face: 

Je t'invite à être patient et à réessayer de temps en temps, ça peut prendre quelques jours voire semaines

**Comment by bil44** [comment]
Salut,
Je confirme (même  erreur 404), service Enedis toujours en rade.

**Comment by bokub** [comment]
Hello,
Le consentement est réparé !

**Comment by vanouzbek** [comment]
Impec tout fonctionne bien! Merci pour cette intégration 😊
